File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        boolean boo = false;;
        if(n>m) boo = true;
        if(eq.equals("=")){
            if(n==m) return 1;
            else if((ineq.equals(">")&&boo)||(ineq.equals("<")&&!boo)) return 1;
        }
        else{
            if(ineq.equals(">")&&boo||(ineq.equals("<")&&!boo)) return 1;
        }
        return 0;
    }
}

File: 프로그래머스/0/181934. 조건 문자열/조건 문자열.py
def solution(ineq, eq, n, m):
    if eq=="=":
        if n==m : return 1
        if (ineq==">" and n>m) or (ineq=="<" and n<m):
            return 1
    else:
        if (ineq==">" and n>m) or (ineq=="<" and n<m):
            return 1
    return 0